package.json

```diff
@@ -16,17 +16,29 @@
       "transform-flow-strip-types"
     ]
   },
+  "eslintConfig": {
+    "extends": "ello",
+    "globals": {
+      "AndroidInterface": false,
+      "ENV": false
+    }
+  },
   "scripts": {
     "build": "babel src --out-dir ./",
-    "build:watch": "babel src --out-dir ./ --watch"
+    "build:watch": "babel src --out-dir ./ --watch",
+    "flow": "flow",
+    "lint": "eslint src test"
```

Comment:
It doesn't. And the test directory is coming later today I hope. Different PR tho.